Automattic/wp-calypso
# Reader: goodbye FeedPostStore

part of the reader reduxification effort https://github.com/Automattic/wp-calypso/issues/10984.

Now that 99% of the functionality from `FeedPostStore` has been moved to redux, this PR attempts to delete the unused code and port over that last 1%  -- I'm looking at you `markSeen`!

**To Test**
- open a stream, click a post, searching for a post. click recs and combined cards.
- open up RelatedPosts and click into more RPs.  Especially for sites like Time and Uproxx
- use the `at` link and click an update pill. then click a gap
- ensure `markSeen` is still working. look for the `reader_pageview` stat
